src/index.js

```diff
@@ -1,118 +1,35 @@
-    };
-
-    mytodoItemLeftTurn.appendChild(myTodoInput);
-
-    const mylistItemdtails = document.createElement('p');
-    mylistItemdtails.classList.add('display-row');
-    if (Todo.completed) {
-      mylistItemdtails.classList.add('border-line');
-    } else {
-      mylistItemdtails.classList.remove('border-line');
-    }
-    mylistItemdtails.innerText = Todo.description;
-    mytodoItemLeftTurn.appendChild(mylistItemdtails);
-
-    const InpuTingmyTodo = document.createElement('input');
-    InpuTingmyTodo.className = 'show-none';
-    InpuTingmyTodo.type = 'text';
-    InpuTingmyTodo.value = Todo.description;
-    InpuTingmyTodo.addEventListener('key', (e) => {
-      if (e.key === 'Enter') {
-        myTodoItem.classList.toggle('back-view');
-        updateTaskTodo(Todo, indexx, e.target.value);
-        showAllMyTask();
-      }
-    });
-    mytodoItemLeftTurn.appendChild(InpuTingmyTodo);
-
-    myTodoItem.appendChild(mytodoItemLeftTurn);
-
-    const mydeleteIcon = document.createElement('span');
-    mydeleteIcon.className = 'show-none';
-    mydeleteIcon.innerHTML = 'deleting';
-    mydeleteIcon.addEventListener('click', () => {
-      deletingMyTodoItem(Todo, indexx);
-      showAllMyTask();
-    });
-    myTodoItem.appendChild(mydeleteIcon);
-
-    const myvertS = document.createElement('span');
-    myvertS.className = 'material-symbols-outlined';
-    myvertS.innerHTML = 'more_vert';
-    myvertS.addEventListener('click', () => {
-      myvertS.className = 'show-none';
-      mydeleteIcon.className = 'material-symbols-outlined';
-
-      mylistItemdtails.className = 'show-none';
-      InpuTingmyTodo.className = 'display-row';
-      myTodoItem.classList.toggle('back-view');
-      InpuTingmyTodo.focus();
-    });
-    myTodoItem.appendChild(myvertS);
-    myTasklistTodo.appendChild(myTodoItem);
-
-    const emptyMyList = document.getElementById('clearMylist');
-    emptyMyList.addEventListener('click', () => {
-      deleteAllmyCompleted(Todo);
-      showAllMyTask();
-    });
+const arrayOfObjects = [
+  {
+    description: 'Wash dishes',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Clean the entire building',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Doing laundry',
+    completed: false,
+    index: 2,
+  },
+];
+
+const myTodoListProjectone = document.querySelector('.dailyTaskOfTodO');
+
+const myToDoComplextion = () => {
+  arrayOfObjects.forEach((taskOfTodo) => {
+    const list = document.createElement('li');
+    list.className = 'mylistItem';
+    list.innerHTML = `<div class="showingMycontent">
+                        <input type="mycheckbox" id="${taskOfTodo.index}" class="check">
+                        <p class="description">${taskOfTodo.description}</p>
+                        <p class="myListholder"><i class="fa-solid fa-ellipsis-vertical"></i></p>
+                    </div><br>
+                    <hr>`; myTodoListProjectone.appendChild(list);
```

Comment:
- [ ] Hey @citec-47, You have well created all the other things as mentioned by the previous reviewer, however, the webpack config does not allow the project to run, see the image below and fix your `start script` to help us check how the project is working.

<img width="528" alt="1" src="https://user-images.githubusercontent.com/77843050/228513480-78b087e0-3cdc-44bf-a6b3-7e118e0b0c87.png">